docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
       - ${NETHERMIND_IP_HTTP:-127.0.0.1}:${NETHERMIND_PORT_HTTP:-8545}:8545 # JSON-RPC
       - ${NETHERMIND_IP_ENGINE:-127.0.0.1}:${NETHERMIND_PORT_ENGINE:-8551}:8551 # ENGINE-API
     command: |
-      --config=${ETH2_NETWORK:-holesky}
+      --config=${NETWORK:-mainnet}
```

Comment:
Could you please leave the default on `holesky` instead? 

This repo is by default on Holesky, and copying `.env.sample.mainnet` makes it run on `mainnet`.

Reply:
Sorry my bad I was modifying an old version. Done!